instructor/common.py

```diff
@@ -723,6 +729,7 @@ def western_blot_band_size(request):
             'formset': formset,
             'types_selected': types_selected,
             'assignment_name': assignment.name,
+            'num_error': json.dumps(error),
```

Comment:
Can you rename num_error to has_error or something to that effect, since it's a bool?